Reyes_JohnRuiz/index.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" /> 
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Reyes John Ruiz Page</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <h1>REYES JOHN RUIZ</h1>
+    </header>
+
+    <main>
+      <section>
+        <h2>My interests and hobbies</h2>
+        <ul>
+          <li>Playing Computer Games</li>
+          <li>Jogging</li>
+          <li>Learning new chords to my guitar</li>
+          <li>Little bit workout</li>
+          <li>Dancing</li>
+        </ul>
+      </section>
+
+      <section>
+        <h2>My works</h2>
+        <ol>
+          <li><a href="./work/business.html">Work 1</a></li>
+          <li><a href="./work/coast.html">Work 2</a></li>
+          <li><a href="./work/info.html">Work 3</a></li>
+          <li><a href="./work/Laws.html">Work 4</a></li>
+          <li><a href="./work/marketing.html">Work 5</a></li>
+          <li><a href="./countries.html">Countries</a></li>
+        </ol>
+      </section>
+
+      <section id="testimonials">
+        <h2>Testimonials</h2>
+        <div class="testimonials">
+          <div>
+            <p>
+              Bergado, Shamma S.J. Monday, April 8, 2024 Working alongside my
+              coding group mate Ruiz has been nothing short of inspiring. His
+              passion for problem-solving and dedication to excellence have made
+              every coding session a rewarding experience.
+            </p>
+          </div>
+
+          <div>
+            <p>
+              Norona, Mary Nicole P. Monday, April 8, 2024 Having a teammate
+              like John, who is a very responsible and hardworking man is one of
```

Comment:
Exceed 80 column

Follow the guidelines:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_0